agent/battle/perception/perception.py

```
@@ -193,9 +193,10 @@ def _count(reco) -> int:
 def _detect_servants(context, img) -> Tuple[ServantState, ...]:
     """检测前排从者的技能可用性。
 
-    识别逻辑：OCR 读取技能按钮上的 CD 节点，若命中数字（剩余回合数）则技能
-    在冷却、不可用；否则（没有任何 CD 数字）技能可用。FGO 里可用技能的按钮上
-    没有 CD 数字，所以"未命中 CD"就是可用的直接信号，不应再标记为未知而被
+    识别逻辑：TemplateMatch 在技能按钮 ROI 内滑动窗口匹配 digits 目录的
```

Ruff (0.16.3)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)